### PR TITLE
Class alien

### PR DESCRIPTION
**Pull request by Raymundogs**
Created a new class, and developed some of the methods it would have, along with an array to display it.

### CODE REVIEW

**Comment by J0c4** [comment]
Hi @Raymundogs @JorgeCaceres-Jala , great work with commits and baby steps, descriptions look good and the changes match according to that description.

Now, there is an issue that in one of the commits, the hidden files were added. You need to revert those similarly to what we did during today's class with @jpgonzales27 
* Remove hidden files with: git rm -r <hidden folder>
* Do a git commit to remove those files
* Do a git rebase to master, so you have .gitignore updated and hidden files are no more tracked

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java [hunk 1]
```diff
@@ -1,8 +1,15 @@
 package org.fundacionjala.at13.spaceinvaders;
 
 public class Main {
-
     public static void main(String[] args) {
-        System.out.println("Space Invaders!!! by AT13");
+        System.out.println("Space Invaders!!! by AT13"+"\n");
+        Matrix spaceNew = new Matrix();
+        // Just a couple of test
```
Comment (JessickaMoya) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Main.java:7]: I think you should erase the commented code

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Alien.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    public String moveDown(){
+        return "Alien moved down";
+    }
+
+    public String shoot(){
+        return "Alien shooted a bullet";
+    }
+
+    public String die(){
```
Comment (GustavoZHA) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Alien.java:43]: Maybe change class die string to boolean, if it dead, return true.

File: src/main/java/org/fundacionjala/at13/spaceinvaders/Alien.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    public String moveLeft(){
+        return "Alien moved to the left";
+    }
+
+    public String moveDown(){
+        return "Alien moved down";
+    }
+
```
Comment (JRojas999) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/Alien.java:38]: It is more convenient that the type of data returned is integer in moveRight, moveLet and moveDown methods

**Pull request merged**